test/AngularSpec.js

```diff
@@ -228,6 +228,18 @@ describe('angular', function() {
       }
     });
 
+    it('should handle Blob objects', function() {
+      if (typeof Blob !== 'undefined') {
+        var src = new Blob(['foo'], {type: 'bar'});
+        var dst = copy(src);
+
+        expect(dst).not.toBe(src);
+        expect(dst.size).toBe(3);
+        expect(dst.type).toBe('bar');
+        expect(isBlob(dst)).toBe(true);
```

Comment:
Have you checked if this test perhaps passes and gives a false positive without the applied `copyType()` patch? I seem to recall when running into this issue first that the cloned object was getting reported as a Blob even though it was not actually one, and I could only detect the difference by attempting to make a call to one of its specific functions like `slice()` or by attempting to read the blob's content by using the `FileReader` interface (either of which then failed in some internal implementation, in different ways depending on the browser used).